README.markdown

```diff
@@ -5,15 +5,23 @@ This project adds promises to Apple’s CoreLocation framework.
 ## CocoaPods
 
 ```ruby
-pod "PromiseKit/CoreLocation", "~> 6.0"
+pod "PromiseKit/CoreLocation", "~> 3.1.2"
```

Comment:
this is incorrect I believe. CoreLocation is a subspec and thus the version is tied to the PromiseKit version.

Reply:
Reverted.